frameworks/kafka/universe/config.json

```diff
@@ -111,8 +101,24 @@
                       "default": "__dcos_base64__kafka_keytab"
                     }
                   }
+                },
+                "tls": {
+                  "description": "TLS settings",
+                  "type": "object",
+                  "properties": {
+                      "enabled": {
+                        "description": "Enable TLS support (requires Enterprise DC/OS running in permissive or strict security mode)",
+                        "type": "boolean",
+                        "default": false
+                      },
+                      "allow_plaintext": {
+                        "description": "Keep the plaintext port open when the TLS is enabled. (Allows clients to connect using a non-encrypted connection and takes effect only if the TLS is enabled.)",
```

Comment:
The frameworks have multiple variants of this `description`. Which is the best one to use?